release-tools/mktarball.sh

```diff
@@ -10,7 +10,7 @@ RELEASEDATE=`date "+%B %d, %Y"`
 
 # Copy bits in
 mkdir ${DESTDIR} ${DESTDIR}/autotools
-cp scrypt_platform.h main.c FORMAT COPYRIGHT ${DESTDIR}
+cp scrypt_platform.h main.c FORMAT COPYRIGHT get-version.sh ${DESTDIR}
```

Comment:
Why?  If we're building a release tarball, it won't have a `.git` directory to extract a version from...